src/amp/components/topMeta/TopMetaNews.tsx

```diff
@@ -63,7 +63,9 @@ export const TopMetaNews: React.FC<{
     articleData: ArticleModel;
     adTargeting?: AdTargeting;
 }> = ({ articleData, adTargeting }) => {
-    const {branding} = articleData.commercialProperties[articleData.editionId];
+    const { branding } = articleData.commercialProperties[
+        articleData.editionId
+    ];
```

Comment:
This formatting makes me cry, but it makes sense.

Reply:
#AutomaticLinter :)